### PR TITLE
SoundCloud: Work around the broken tokens.

### PR DESCRIPTION
**Pull request by jagtalon**
Things like "soundcloud ray bradbury" seem to work

### CODE REVIEW

File: lib/DDG/Spice/SoundCloudResult.pm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+use strict;
+use DDG::Spice;
+
+description "Displays audio from SoundCloud";
+name "SoundCloud";
+primary_example_queries "soundcloud MGMT";
+secondary_example_queries "sc the new yorker";
+topics "entertainment", "music";
+category "entertainment";
```
Comment (russellholt) [inline lib/DDG/Spice/SoundCloudResult.pm]: don't need all this stuff

File: share/spice/sound_cloud/sound_cloud.js [hunk 2]
```diff
@@ -2,75 +2,75 @@
...
-
-        var SOUNDCLOUD_CLIENT_ID = 'df14a65559c0e555d9f9fd950c2d5b17',
-            script = $('[src*="/js/spice/sound_cloud/"]')[0],
+        var script = $('[src*="/js/spice/sound_cloud/"]')[0],
             source = $(script).attr("src"),
-            query = source.match(/sound_cloud\/([^\/]*)/)[1],
-
+            query = source.match(/sound_cloud\/([^\/]*)/)[1];
+        
+        $.getJSON("/js/spice/sound_cloud_result/" + query, function(api_result) {
```
Comment (russellholt) [inline share/spice/sound_cloud/sound_cloud.js]: don't name it `api_result`, that's confusing with the above. change one or the other

**Comment by jdorweiler** [comment]
:+1:

**Comment by jdorweiler** [comment]
Tested and everything works.

**Pull request merged**